pkg/controller/node/nodecontroller.go

```diff
@@ -198,6 +198,7 @@ func NewNodeController(
 		forcefullyDeletePod:       func(p *api.Pod) error { return forcefullyDeletePod(kubeClient, p) },
 		nodeExistsInCloudProvider: func(nodeName string) (bool, error) { return nodeExistsInCloudProvider(cloud, nodeName) },
 		computeZoneStateFunc:      ComputeZoneState,
+		zoneStates:                map[string]zoneState{},
```

Comment:
Yeah. What I had in mind is whether it won't cause entering some state initially. But discussed offline and it seems it doesn't cause any issues.